packages/elements-core/src/hoc/withQueryClientProvider.tsx

```diff
@@ -7,7 +7,8 @@ const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       retry: false,
-      staleTime: 15 * 1000,
+      refetchOnWindowFocus: false,
+      staleTime: Infinity,
```

Comment:
this will break editing.

When users add new article for example in studio, and go back to platform, they will never see it.

Reply:
This was just an experiment. Changing it back!